aviary/subsystems/propulsion/hamilton_standard.py

```diff
@@ -722,25 +723,29 @@ def compute(self, inputs, outputs):
                     ang_len = ang_arr_len[kdx]
                     BLL[kdx], run_flag = _unint(
                         CP_Angle_table[idx_blade][kdx][:ang_len], Blade_angle_table[kdx], CP_Eff)  # blade angle at baseline point for kdx
-                    CTT[kdx], run_flag = _unint(
-                        Blade_angle_table[kdx], CT_Angle_table[idx_blade][kdx][:ang_len], BLL[kdx])  # thrust coeff at baseline point for kdx
+                    try:
+                        CTT[kdx], run_flag = _unint(
+                            Blade_angle_table[kdx], CT_Angle_table[idx_blade][kdx][:ang_len], BLL[kdx])  # thrust coeff at baseline point for kdx
+                    except IndexError:
+                        raise om.AnalysisError(
+                            "interp failed for CTT in hamilton_standard.py")
```

Comment:
what is CTT? What does that stand for?

Reply:
Changed to `"interp failed for CTT (thrust coefficient) in hamilton_standard.py")`

Reply:
CTT is an intermediate or temporary CT (thrust coefficient) in the middle of computation.